lib/src/stack/coap_blockwise_layer.dart

```diff
@@ -286,45 +254,37 @@ class CoapBlockwiseLayer extends CoapAbstractLayer {
         // transfer. Thus, deliver it.
         _clearBlockCleanup(exchange);
         super.receiveResponse(nextLayer, exchange, response);
-      } else {
-        _log!
-            .info('Response has Block2 option and is therefore sent blockwise');
       }
     }
 
     final block2 = response.block2;
     if (block2 != null) {
-      _log!.info('Blockwise block2 - response acknowledges block: $block2');
       var status = _findResponseBlockStatus(exchange, response);
-      _log!.info('Blockwise exchange block2 status is - $status');
       final blockStatus = CoapBlockOption(optionTypeBlock2);
-      blockStatus.rawValue = status.currentNUM;
+      blockStatus.rawValue = status.currentNUM!;
```

Comment:
I think the type of `currentNUM` could be changed from `int?` to `int`.

Reply:
Fixed: https://github.com/shamblett/coap/pull/76/commits/e3d2bff4bf1ebcf93e8a832e13d29aa9c4a2f3f7